lms/templates/student_account/login.underscore

```diff
@@ -11,7 +11,8 @@
         <h4 class="message-title"><%- gettext("Password Reset Email Sent") %></h4>
         <div class="message-copy">
             <p>
-                <%- gettext("We've sent instructions for resetting your password to the email address you provided.") %>
+                <%- gettext("We have sent an email message with password reset instructions to the email address you provided.  If you do not receive this message, ") %>
```

Comment:
You should not use language fragments with ``gettext`` because some languages might not include the fragments in the same order. You can see an explanation of this here:

http://edx-developer-guide.readthedocs.io/en/latest/conventions/internationalization/i18n.html#general-internationalization-rules

Instead you should look into using ``HtmlUtils.interpolate`` to create the full sentence. If you are not familiar with this, you can read about it here:

http://edx-developer-guide.readthedocs.io/en/latest/conventions/safe_templates.html#safe-javascript-files